ethereum/contracts/CappedMintableToken.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity >=0.4.22 <0.6.0;
+import "@openzeppelin/contracts/token/ERC20/ERC20Mintable.sol";
+
+contract CappedMintableToken is ERC20Mintable {
+    uint256 private cap;
+
+    constructor(uint256 _cap) public {
+        require(_cap > 0, "cap amount must be higher than 0");
+        cap = _cap;
+    }
+
+    function getCap() public view returns(uint256) {
```

Comment:
`ERC20Capped` already implements ` function cap() public view returns (uint256)` which is doing the same thing.